CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## vNext
+
+- Support query params for `GET /accounts` endpoint
```

Comment:
Which query params are added? Just for pagination, no?

Reply:
Also account status

Reply:
AccountId and Status